src/main/java/seedu/address/ui/ContactDetails.java

```diff
@@ -0,0 +1,91 @@
+package seedu.address.ui;
+
+import java.util.logging.Logger;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.model.person.Person;
+
+/**
+ * Controller for the contact details panel.
+ */
+public class ContactDetails extends UiPart<Region> {
+
+    private static final String FXML = "ContactDetails.fxml";
+    private final Logger logger = LogsCenter.getLogger(ContactDetails.class);
+
+    private Person person;
+
+    @FXML
+    private HBox contactDetailPane;
+
+    @FXML
+    private Label name;
+
+    @FXML
+    private Label phoneNo;
+
+    @FXML
+    private Label email;
+
+    @FXML
+    private Label address;
+
+    @FXML
+    private Label notes;
+
+    @FXML
+    private VBox notesList;
+
+    /**
+     * Creates a {@code ContactDetailPannel} with the given {@code Person} information.
```

Comment:
I think this is a typo
```suggestion
     * Creates a {@code ContactDetailPanel} with the given {@code Person} information.
```